dandi/dandiapi.py

```diff
@@ -1,3 +1,29 @@
+"""
+Example code for downloading all assets from all published Dandisets that have
+"two-photon" in their ``metadata.measurementTechnique[].name``:
+
+.. code:: python
+
+    from operator import attrgetter
```

Comment:
```suggestion
```
since no longer used.

Please make this code snippet/docstring  doc tested so we could ensure it stays valid.

With that let's finalize this PR and let's get it merged.

Reply:
@yarikoptic The code downloads assets.  How is that supposed to be safely doctested (to say nothing of how long it could take to run the code)?

Reply:
d'oh , right.  Lets may be comment out the `download` line (with a comment to use it if download is desired) and just print metadata?  I just feel that having at least majority of the docstring sample tested would be of great benefit long term

Reply:
@yarikoptic Doctests work by comparing the code's output to the output given in the doctest.  Aside from the fact that most Dandiset metadata is multiple lines long even when not formatted, we would have to update the doctest whenever a new version is published.

Reply:
could we use ` # doctest:+ELLIPSIS` and have `...` to swallow all output?

Reply:
@yarikoptic That works, but it adds about 50 seconds to the total test runtime.

Reply:
that is indeed notable.  Could we may be run such a full sweep with doc-tests only on cron (e.g. daily)?

Reply:
@yarikoptic Done.

Reply:
Looks good to me.  Time to undraft and merge?